### PR TITLE
Added prettier config and applied to all files.

### PR DESCRIPTION
**Pull request by Torbaz**
Not nessesary change.
Personal preference if you want to use prettier or keep manual formatting.

### CODE REVIEW

**Comment by alex-ong** [comment]
Are you using an automated tool for this? By automated i mean literally running one command.

In python land there is `black .` and `pylint`. Are you using an equivalent? Prettifiying the code is important, it is much cleaner. But moving forward it needs to be automated; so i suggest rolling that functionality into this PR.

**Comment by Torbaz** [comment]
I reformatted all the files and added a config file using "prettier" https://prettier.io/, which formats based on the configuration on a per project level. Anyone who installs and sets up prettier will automatically follow the rules layed out in the config file.

**Comment by alex-ong** [comment]
I suggest adding documentation for installing prettier as a precommit hook then; this could be inside developerment.md or readme.md

File: .prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+	"trailingComma": "es5",
+	"semi": true,
+	"singleQuote": true,
+	"useTabs": true
+}
```
Comment (timotheeg) [inline .prettierrc.json:6]: From the style that I have, I think these 3 should be set too:
```
"bracketSpacing": true,
"arrowParens": "avoid",
"quoteProps": "consistent"
```

**Review by timotheeg — COMMENTED** [review]
Wow! Thanks for that PR!

I know the whole point of opinionated formatter is to stop having pointless discussions on code style, so we can all focus on content instead... But I'm an OTBS guy, so boy, do I not like this:
```
} else {
```
as opposed to
```
}
else {
```

😭 

And I also love lots of breathing room in my code with creative multi-line formatting and/or vertical alignment, and the stupid formatters always kill my (obviously biased) beautiful stuff 😭 

Oh well... I will get used to it I guess, it's for the greater good!

One more important thing though: `opencv.js` is imported as-is from the published version, and it should never be edited. Let's add a [.prettierignore file](https://prettier.io/docs/en/ignore.html#ignoring-files) and list that file inside.

And yes, agreed with @alex-ong: either in this PR, or shortly thereafter, we should set a pre-commit hook so PRs are guaranteed to have a clean format from day 1.

**Pull request merged**